wandb/sdk/wandb_config.py

```diff
@@ -94,6 +94,9 @@ def __setitem__(self, key, val):
         if self._callback:
             self._callback(key=key, val=val, data=self._as_dict())
 
+    def items(self):
+        return self._items.items()
```

Comment:
I think we might want to hide some items.  namely anything with the key `_wandb`, but it looks like elsewhere we are hiding anything beginning with "_"